application/views/admin/globalsettings/_emailTestConfirmation.php

```diff
@@ -0,0 +1,38 @@
+<div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal">&times;</button>
+    <div class="h3 modal-title"><?php eT("Test Email Settings"); ?></div>
+</div>
+<div class="modal-body">
+    <p><?php eT("Test email will be sent to:"); ?> <strong><?= $testEmail; ?></strong></p>
+    <div id="settingschangedwarning" class="jumbotron message-box message-box-error" style="display: none;">
+        <p class="text-warning"><?= gT("There seems to be some changes in the settings which were not saved yet. These changes will be disregarded by the test procedure."); ?></p>
+    </div>
+</div>
+<div class="modal-footer">
+    <a class="btn btn-primary btn-ok" href="<?php echo \Yii::app()->createUrl('admin/globalsettings', array("sa"=>"sendTestEmail"));?>"><span class='fa fa-check'></span>&nbsp;<?php eT("Send Email"); ?></a>
+    <button type="button" class="btn btn-danger" data-dismiss="modal">
+        <?php eT("Close"); ?>
+    </button>
+</div>
+
+<script>
```

Comment:
I don't understand usage of this script ?

Reply:
This scripts identfies if any setting has been changed and not saved. Then it alerts the user about that

Reply:
Oh, it's on **current** global setting … no way to add a dedicated page ? In my opinion : it's really the best way.

Reply:
The idea is to test the settings. How do you picture it? This is already done. The idea was to port it and adapt it. Not re do it.

Reply:
> The idea was to port it and adapt it

? It don't exist in 3.X , then it's a new feature , no ?

Reply:
It is a port from a PR already made